### PR TITLE
Contributing

### PR DESCRIPTION
**Pull request by KenCarvAPI**
Introduction of a 'feedback to API3 docs' page 
Review and build of 'contribute to API3 DAO' section

### CODE REVIEW

**Comment by wkande** [comment]
@KenCarvAPI 

You have some heading tags that do not make sense.

- (Heading1) should only be used once per page at the top of the page: # {{$frontmatter.title}}
- Heading2 (as many as needed) must be after a (# Heading1). These really separate the page into new subject matter related to (# Heading1)
- Heading3 can only be nested after a (# Heading2) and are related to the subject matter of the proceeding (## Heading2)

File: docs/explore/introduction/contributing.md [hunk 1]
```diff
@@ -14,80 +14,145 @@ tags:
...
 
-API3 is open to any proposals and improvements. Please feel free to contribute.
+We think critical infrastructure, like API3 DAO's first-party oracles, should
+implement open and transparent contributor processes to align to our vision of
+being a programmatically driven protocol.
+
+To support DAO governance and inform voters, all teams publish a public proposal
+at regular intervals, meaning strategy & operations are accessible on the
+[API3 Governance Forum](https://forum.api3.org/) for all to understand.
+Additionally, teams generally update the community with retrospective reports.
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:26]:
```suggestion
Additionally, teams generally update the community with ongoing reports.
```

File: docs/explore/introduction/contributing.md [hunk 2]
```diff
@@ -14,80 +14,145 @@ tags:
 
 <FlexStartTag/>
 
-# {{$frontmatter.title}}
+# {{$frontmatter.title}} 
 
-API3 is open to any proposals and improvements. Please feel free to contribute.
+We think critical infrastructure, like API3 DAO's first-party oracles, should
+implement open and transparent contributor processes to align to our vision of
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:20]:
```suggestion
implement open and transparent contributor processes to align with our vision of
```

File: docs/explore/introduction/contributing.md [hunk 3]
```diff
@@ -14,80 +14,145 @@ tags:
...
+at regular intervals, meaning strategy & operations are accessible on the
+[API3 Governance Forum](https://forum.api3.org/) for all to understand.
+Additionally, teams generally update the community with retrospective reports.
+
+## Types of contributors
+
+Community participants within a Decentralized Autonomous Organization (DAO) have
+an active role in shaping the future of the project. Our aim is to simplify the
+process for those who wish to participate, providing predefined methods for
+engagement. Generally these take forum in three routes:
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:33]:
```suggestion
engagement. Generally these take three forms:
```

File: docs/explore/introduction/contributing.md [hunk 4]
```diff
@@ -14,80 +14,145 @@ tags:
...
+take on the job, complete it, and subsequently receive the designated grant as a
+reward retrospectively.
+
+### Submit your own proposal
+
+We welcome proposals that bring new ideas or sub-teams to the DAO. Generally
+engaging the DAO through our community in a constructive manner will help
+sentiment check a proposal and help shape it to align to current contributors.
+If you are proposing an initiative that would require collaboration with a
+current sub team, we suggest co-ordinating proactively and outlining how you
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:62]:
```suggestion
current sub team, we suggest coordinating proactively and outlining how you
```

File: docs/explore/introduction/contributing.md [hunk 5]
```diff
@@ -14,80 +14,145 @@ tags:
...
+will work together.
+
+# Put forward your ideas to the API3 DAO
+
+Regardless of the contributor's skill, API3's vision is for participation to be
+as frictionless as possible. Individuals with a relevant background can
+contribute to the project in a flexible manner, replicating similar
+freelancer/consultancy type figures commonly seen.
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:70]:
```suggestion
freelancer/consultancy type models commonly seen.
```

File: docs/explore/introduction/contributing.md [hunk 6]
```diff
@@ -14,80 +14,145 @@ tags:
...
+
+## Technical contributors
+
+For technical folks this could be through development, integrations, writing
+documentation, etc. Although the general challenge is that one needs to be able
+to use a solution proficiently before being able to improve it. If this applies
+we suggest digesting the get started materials.
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:77]: A link would be helpful for "getting started"

File: docs/explore/introduction/contributing.md [hunk 7]
```diff
@@ -14,80 +14,145 @@ tags:
...
-make sure to ask any questions you have on
-[Discord<ExternalLinkImage/>](https://discord.gg/qnRrcfnm5W).
+We recommend spending some time learning about API3 DAO's oracle technology
+before looking to contribute from an engineer perspective. We welcome you to ask
+any questions on [Discord<ExternalLinkImage/>](https://discord.gg/qnRrcfnm5W).
 
+- Read the entirety of these docs. Take notes and cross-reference, but do not
+  expect to understand everything. Feel free to ask questions.
+- Visit the [API3 GitHub<externalLinkImage/>](https://github.com/api3dao) page
+  and find everything about.. everything. Explore the many repositories to gain
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:94]: I suggest rewording this line.

File: docs/explore/introduction/contributing.md [hunk 8]
```diff
@@ -14,80 +14,145 @@ tags:
...
-- To get a general grasp of the project, read the
-  <a href="/api3-whitepaper-v1.0.3.pdf" target="_api3-whitepaper">API3
-  Whitepaper<ExternalLinkImage/></a> and our
-  [Blog Posts on Medium<ExternalLinkImage/>](https://medium.com/api3).
+## Non-technical contributors
 
-- Read the entirety of these docs. Take notes and cross-reference, but do not
-  expect to understand everything. Feel free to ask questions.
+Marketers, creatives and community builders can contribute to DAO subteams with
+ad-hoc projects or to support areas that are lacking focus. The challenge here
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:114]: I'd suggest rewording 'are lacking focus'

File: docs/explore/introduction/contributing.md [hunk 9]
```diff
@@ -14,80 +14,145 @@ tags:
...
 
-- the whitepaper
-- source code
-- documentation
-- usage examples
+## This can be submitted in a variety of way
 
-Please feel free to contribute to any API3 GitHub repositories.
+- Join the API3 [community<ExternalLinkImage/>](https://discord.gg/qnRrcfnm5W)
+and submit within the ecosystem contributor request channel.
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:142]: This channel doesn't exist?

File: docs/explore/introduction/contributing.md [hunk 10]
```diff
@@ -14,80 +14,145 @@ tags:
...
-- source code
-- documentation
-- usage examples
+## This can be submitted in a variety of way
 
-Please feel free to contribute to any API3 GitHub repositories.
+- Join the API3 [community<ExternalLinkImage/>](https://discord.gg/qnRrcfnm5W)
+and submit within the ecosystem contributor request channel. 
+- Submit an
+overview onto the [API3 forums](https://forum.api3.org/).
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:144]: I would suggest using "sentiment check" instead of overview, and/or linking to https://docs.api3.org/guides/dao-members/proposals.html#proposal-creation

File: docs/explore/introduction/documentation-feedback.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+a fork is always welcome.
+
+You can also see the [Contributing](/docs/explore/introduction/contributing.md) overview on how to engage further.
+
+### Join the API3 testers guild
+
+API3 Testers Guild is a community of professionals involved to testing API3
+products & services. The Guild often conducts workshops, webinars, and
+conferences to foster a dynamic and up-to-date understanding of testing in the
+evolving tech landscape.
```
Comment (dcroote) [inline docs/explore/introduction/documentation-feedback.md:66]: Are there any links or more info for joining this?

**Review by dcroote — CHANGES_REQUESTED** [review]
Nice start- I have several questions and suggestions.

Please also spell check both documents, there are a number of typos.

@wkande - once updated and ready, this PR will need to be squash merged because of the git history

File: docs/explore/introduction/contributing.md (hunk 6 above)
Comment (KenCarvAPI) [reply docs/explore/introduction/contributing.md:77]: updated

File: docs/explore/introduction/contributing.md (hunk 8 above)
Comment (KenCarvAPI) [reply docs/explore/introduction/contributing.md:114]: Re worked area a little, thanks

File: docs/explore/introduction/documentation-feedback.md (hunk 11 above)
Comment (KenCarvAPI) [reply docs/explore/introduction/documentation-feedback.md:66]: There will be soon, it's not firmed up yet. It will be added to the ecosystem gitbook - can review and update here. 

Should I resolve this?

File: docs/explore/introduction/contributing.md [hunk 12]
```diff
@@ -110,18 +113,19 @@ any questions on [Discord<ExternalLinkImage/>](https://discord.gg/qnRrcfnm5W).
 
 ## Non-technical contributors
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:114]: I think you want 3 # here

File: docs/explore/introduction/contributing.md [hunk 13]
```diff
@@ -110,18 +113,19 @@ any questions on [Discord<ExternalLinkImage/>](https://discord.gg/qnRrcfnm5W).
...
-Marketers, creatives and community builders can contribute to DAO subteams with
-ad-hoc projects or to support areas that are lacking focus. The challenge here
-is accurately scoping, forming a brief proposed solution method, and then
+Marketers, creatives and community builders can contribute to DAO with ad-hoc
+projects or to support specific areas within subteams. The challenge here is
+accurately scoping, forming a brief proposed solution method, and then
 delivering in alignment with other involved contributors.
 
-Contirbutions may look like:
+Amoung many other types, contirbutions may look like:
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:121]: Please use spellcheck

File: docs/explore/introduction/contributing.md [hunk 14]
```diff
@@ -132,23 +136,27 @@ We appreciate the individual nature of contributing and these frameworks are
...
-## Ecosystem fast bounties & retropective grants
+### Ecosystem fast bounties & retropective grants
 
-The API3 Ecosystem team has a dedicated pool of funds to support micro-grants from community contributors. This can be retrospective or pre-planned, but has been created to make rewarding contribution frictionless and simple. 
+The API3 Ecosystem team has a dedicated pool of funds to support micro-grants
+from community contributors. This can be retrospective or pre-planned, but has
+been created to make rewarding contribution frictionless and simple.
 
-## This can be submitted in a variety of way
+### This can be submitted in a variety of way
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:145]: I suggest rewording this section title

File: docs/explore/sidebar.js [hunk 15]
```diff
@@ -31,7 +31,14 @@ module.exports = [
 
       //{ text: 'Blog Posts on Medium', link: '/explore/introduction/blog' },
       //{ text: 'Jobs', link: '/explore/introduction/jobs' },
-      { text: 'Contributing', link: '/explore/introduction/contributing' },
+      {
+        text: 'Contributing to the DAO',
+        link: '/explore/introduction/contributing',
+      },
+      {
+        text: 'Feedback to API3 Docs',
```
Comment (dcroote) [inline docs/explore/sidebar.js:39]: Perhaps "API3 Docs Feedback"

File: docs/explore/introduction/contributing.md [hunk 16]
```diff
@@ -16,78 +16,151 @@ tags:
...
+with other contributors necessary to accomplish the deliverables and (3) the
+amount of API3 tokens requested.
 
-This page will be updated as more materials are produced. Please feel free to
-create
-[GitHub issues<ExternalLinkImage/>](https://github.com/api3dao/vitepress-docs/issues)
-to make suggestions.
+We appreciate the individual nature of contributing and these frameworks are
+flexible, yet linking these activities to the challenges outlined in team
+proposals will support our hollisitic objectives and be more likely supported.
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:137]: spellcheck

File: docs/explore/introduction/contributing.md [hunk 17]
```diff
@@ -16,78 +16,151 @@ tags:
...
 
-## GitHub
+### Ecosystem fast bounties & retropective grants
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:139]: spellcheck

File: docs/explore/introduction/contributing.md [hunk 18]
```diff
@@ -16,78 +16,151 @@ tags:
...
+- Submit an overview and sentiment check onto the
+  [API3 forums](https://forum.api3.org/).
+- Follow the
+  [proposal guidelines](https://docs.api3.org/guides/dao-members/proposals.html#proposal-creation)
+  provided by API3 DAO.
+- Email ecosystem@api3.org with an overview
 
-## Communities
+Within this process we welcome open dialogue and as a DAO recongize that
+external contributors and diversity in thought is a strenght.
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:157]: spellcheck & same on line above

File: docs/explore/introduction/documentation-feedback.md [hunk 19]
```diff
@@ -0,0 +1,70 @@
...
+
+### Submit an issue
+
+We welcome all developers to contribute to the growth and enhancement of API3
+DAO by submitting any issues or suggestions you come across via our issue
+template. Your insight is invaluable in making our technology better.
+
+Please submit to https://github.com/api3dao/vitepress-docs/issues.
+
+::: tip Feedback is valueable
```
Comment (dcroote) [inline docs/explore/introduction/documentation-feedback.md:46]: No 'e' in valuable, also applies elsewhere

**Review by dcroote — CHANGES_REQUESTED** [review]
Looks pretty good, only minor changes left

File: docs/explore/introduction/documentation-feedback.md (hunk 11 above)
Comment (dcroote) [reply docs/explore/introduction/documentation-feedback.md:68]: Should we take this out until there is something concrete then?

Comment (KenCarvAPI) [reply docs/explore/introduction/documentation-feedback.md:68]: kk taken out

File: docs/explore/introduction/documentation-feedback.md [hunk 20]
```diff
@@ -0,0 +1,70 @@
...
+:::
+
+## Contributing to the Technical Documentation
+
+See the [dev documentation](/dev/) for an overview of the API3 technical
+documentation site. It contains instructions on the make-up of the docs and on
+how to clone, setup, and run the project locally. For non-DAO members a PR from
+a fork is always welcome.
+
+You can also see the [Contributing](/docs/explore/introduction/contributing.md)
```
Comment (dcroote) [inline docs/explore/introduction/documentation-feedback.md:60]: 404 currently. 
```suggestion
You can also see the [Contributing](contributing.md)
```

File: docs/explore/introduction/contributing.md [hunk 21]
```diff
@@ -16,78 +16,151 @@ tags:
...
+- Head to the
+  [API3 Community Hub](https://github.com/api3-ecosystem/doc-hub.git) with get
+  started tutorials & demo projects.
+- Go through the issues, PRs, etc. and understand the current focus of the
+  development team by viewing the development board.
+
+### If you are learning about blockchain & oracles
+
+- Read the
+  [getting started<ExternalLinkImage/>](docs/explore/introduction/index.md)
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:105]: 404 currently.
```suggestion
  [getting started<ExternalLinkImage/>](index.md)
```

**Review by dcroote — CHANGES_REQUESTED** [review]
404s need to be fixed, then I think it will be good to go

File: docs/explore/introduction/contributing.md (hunk 21 above)
Comment (wkande) [reply docs/explore/introduction/contributing.md:105]: Use: `/explore/introduction/index.md`

File: docs/explore/introduction/documentation-feedback.md (hunk 20 above)
Comment (wkande) [reply docs/explore/introduction/documentation-feedback.md:60]: Use: `/explore/introduction/contributing.md`

**Comment by wkande** [comment]
I will merge this then pull and fix the links and re-build the indexes.

**Comment by dcroote** [comment]
Thanks @wkande
